chaos.py

```diff
@@ -69,12 +74,17 @@ def main():
 
     api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
 
+    # Api Twitter
+    api_twitter = ta.API_TWITTER(settings.TWITTER_API_KEYS_FILE)
+
     log.info("checking if I crashed before...")
+    ta.Twitter.PostTwitter("checking if I crashed before...", api_twitter.GetApi())
```

Comment:
Do we really want it to be this verbose? I don't think it needs to tweet this out every time it starts up.